core/src/main/java/io/questdb/cutlass/pgwire/PGConnectionContext.java

```diff
@@ -1424,6 +1424,11 @@ private void parse(long address, int len, @Transient SqlCompiler compiler)
             case 'Q':
                 processQuery(msgLo, msgLimit, compiler);
                 break;
```

Comment:
Is this CopyDone message handler required for jdbc or other clients ?

Reply:
this line sends it: https://github.com/questdb/questdb/pull/2061/files#diff-8a715db0fe34387362d99b2bf088facf1faa2c5d0c6192abd0a2e7cd8d1e1fc1R1693

I could not figure out how to use COPY without the `CopyManager` API. Just using `statement.query(copy ...)` does not work for me. I can try whether it works with Postgres.

Reply:
I pushed the new version and I believe it simplified things significantly. also it means LOCAL COPY work as a regular COPY query. Beside the JDBC test I also tested it with a Python Postgres client and it works nicely. 
```python
import psycopg2 as pg
conn = pg.connect(user="admin",
                                  password="quest",
                                  host="localhost",
                                  port="8812",
                                  database="qdb")
cursor = conn.cursor()

cursor.execute('drop table if exists try')
cursor.execute('COPY try FROM "core/src/test/resources/csv/test-numeric-headers.csv"')

cursor.close()
conn.close()
```